packages/client/src/modules/post/containers/Post.spec.js

```diff
@@ -327,18 +328,17 @@ describe('Posts and comments example UI works', () => {
     expect(content.find('.delete-comment')).has.lengthOf(6);
   });
 
-  step('Comment editing works', done => {
+  step('Comment editing works', async done => {
     mutations.editComment = (obj, { input }) => {
       expect(input.postId).to.equal(3);
       expect(input.content).to.equal('Edited comment 2');
       done();
       return input;
     };
-
     const editButtons = content.find('.edit-comment');
     expect(editButtons).has.lengthOf(6);
     editButtons.last().simulate('click');
-
+    editButtons.last().simulate('click');
```

Comment:
Why do we click two times here?

Reply:
When I use a simulate('click') once, the test is fail. I tried several solutions from this article https://github.com/airbnb/enzyme/issues/1201, but they did not help.
I think this problem is related to the version of the "enzyme".

Reply:
@sergei-gilevich Oh, I see